pkix.go

```diff
@@ -142,3 +145,21 @@ func GetCertificateRevocationList(d Depot, name string) (*pkix.CertificateRevoca
 func DeleteCertificateRevocationList(d Depot, name string) error {
 	return d.Delete(depot.CrlTag(name))
 }
+
+// PutTTL puts a new TTL for a given name in the mongo depot.
+func PutTTL(d Depot, name string, expiration time.Time) error {
+	md, ok := d.(*mongoDepot)
+	if !ok {
+		return errors.New("cannot put TTL if depot is not a mongo depot")
+	}
+	return md.PutTTL(name, expiration)
```

Comment:
maybe here we should do a check against the actual expiration of the cert using the ValidityBounds function (in cert.go) to ensure the TTL set does not exceed the actual expiration.

Reply:
Done.